rtos/rtx/TARGET_CORTEX_A/RTX_CM_lib.h

```diff
@@ -50,7 +50,7 @@
 #define _declare_box(pool,size,cnt)  uint32_t pool[(((size)+3)/4)*(cnt) + 3]
 #define _declare_box8(pool,size,cnt) uint64_t pool[(((size)+7)/8)*(cnt) + 2]
 
-#define OS_TCB_SIZE     52
+#define OS_TCB_SIZE     60
```

Comment:
Yep, it is the size of OS_TCB.  It was defiantly too small on the cortex M.  I didn't run the cortex A but I think this one was too small as well.